model/enduser/registry.yaml

```diff
@@ -0,0 +1,42 @@
+groups:
+  - id: registry.enduser
+    type: attribute_group
+    display_name: End User Attributes
+    brief: >
+      Describes information about the end user, which can be used as a subdomain of browser, client, or user domains.
+    attributes:
+      - id: enduser.id
+        type: string
+        deprecated: Replaced by `enduser.anonymous.id` attribute.
+        stability: experimental
+        examples: ['QdH5CAWJgqVT4rOr0qtumf']
+        brief: >
+          Deprecated, use `enduser.anonymous.id` instead.
+      - id: enduser.anonymous.id
+        type: string
+        stability: experimental
+        brief: >
+          Identifier of an anonymous end user who interacts with a system.
+          This identifier may be unique only through best-effort means and does not imply that the user is authenticated to the system.
```

Comment:
meeting notes from semantic convention SIG meeting today:

* is this something that OpenTelemetry instrumentation is planning populate, or is this only something that vendor-specific instrumentation is planning to populate?
* if it's something that OpenTelemetry instrumentation is planning to populate, what would the implementation look like, e.g. would this be stored in a persistent cookie? would it be stamped onto a specific event?
* there is a desire not to add attributes into the semconv repo when only a single vendor has expressed interest in them
* there is also a desire not to add attributes into the semconv repo without having any span/event/metric definitions that use them

the recommendation for next steps was to discuss this in the Client SIG

Reply:
@heyams were those questions addressed during client sig meeting?

Reply:
can you post the answers here?

Reply:
> is this something that OpenTelemetry instrumentation is planning populate

- Yes, this will be added by instrumentations either directly or indirectly (via a psuedo user manager  -- like the Session Manager)

> if it's something that OpenTelemetry instrumentation is planning to populate, what would the implementation look like, e.g. would this be stored in a persistent cookie? would it be stamped onto a specific event?

- It will need something like the SessionManager implementation to "manager" the lifecycle of this value, some environments would be just to create a simple random value for App start (like Android) while in a browser which is stateless then the "user manager" would most likely use cookies (but it could also use session storage)

> there is a desire not to add attributes into the semconv repo when only a single vendor has expressed interest in them

- This will not be single vender specific, but it is highly RUM specific

> there is also a desire not to add attributes into the semconv repo without having any span/event/metric definitions that use them

- It should be available both Spans and Logs, while it could be available for Metrics it's cardinality (because its a random value) should not necessarily be used for metrics...

Reply:
thanks! a couple of followup questions:

* if/when entity (or mutable resource) attributes are available, do you see `enduser.pseudo.id` as one of those?
* are there any specific span or event semantic conventions that we can add `enduser.pseudo.id` to? or is `enduser.pseudo.id` hopefully just an entity (or mutable resource) attribute?

Reply:
I believe we discussed these in the client SIG. I will wait for @MSNev's response.

Reply:
- If/when mutable resources attributes (whether they are entities or [shorter lived lifespan resources](https://github.com/open-telemetry/opentelemetry-specification/pull/4316) ) then this (and session) could possibly by represented their.

> are there any specific span or event semantic conventions that we can add enduser.pseudo.id to?

No, not specifically as this really is just additional context details.